src/components/CertificateVerifyBlock/CertificateVerifyBlock.tsx

```diff
@@ -1,30 +1,42 @@
 import { VerificationFragment, VerificationFragmentStatus } from "@govtechsg/oa-verify";
+import { getData, v2, WrappedDocument } from "@govtechsg/open-attestation";
 import { OpencertsRegistryVerificationFragmentData } from "@govtechsg/opencerts-verify";
 import React, { useState } from "react";
 import { icons } from "../ViewerPageImages";
 import { DetailedCertificateVerifyBlock } from "./DetailedCertificateVerifyBlock";
 
 type DnsFragment = { status: VerificationFragmentStatus; location?: string };
 
-export const getIdentityVerificationText = (verificationStatus: VerificationFragment[]): string => {
+export const getIdentityVerificationText = (
+  verificationStatus: VerificationFragment[],
+  document: WrappedDocument<v2.OpenAttestationDocument>
+): string => {
+  const data = getData(document);
   const registryFragment = verificationStatus.find((status) => status.name === "OpencertsRegistryVerifier");
-  const dnsFragment = verificationStatus.find((status) => status.name === "OpenAttestationDnsTxt");
+  const dnsFragment = verificationStatus.find((status) => status.name === "OpenAttestationDnsTxtIdentityProof");
+  const dnsDidFragment = verificationStatus.find((status) => status.name === "OpenAttestationDnsDidIdentityProof");
```

Comment:
not sure if u want to be more explicit here on variable naming
```suggestion
  const dnsTxtFragment = verificationStatus.find((status) => status.name === "OpenAttestationDnsTxtIdentityProof");
  const dnsDidFragment = verificationStatus.find((status) => status.name === "OpenAttestationDnsDidIdentityProof");
```

Reply:
done